README.md

```diff
@@ -2,6 +2,7 @@
 
 # `svc-telemetry`
 
+![GitHub stable release (latest by date)](https://img.shields.io/github/v/release/Arrow-air/svc-telemetry?sort=semver&color=green)
 ![GitHub release (latest by date including pre-releases)](https://img.shields.io/github/v/release/Arrow-air/svc-telemetry?include_prereleases)
 ![Sanity Checks](https://github.com/arrow-air/svc-telemetry/actions/workflows/sanity_checks.yml/badge.svg?branch=develop)
 ![Rust Checks](https://github.com/arrow-air/svc-telemetry/actions/workflows/rust_ci.yml/badge.svg?branch=develop)
```

Comment:
Should be Python PEP 8

Reply:
Are you referring to the "Sanity Checks" label?

Reply:
Fixed: https://github.com/Arrow-air/svc-telemetry/pull/14/commits/520125b25c771097232030ac12d047a2fa7193f8